catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -244,6 +300,68 @@ private CqlQueryResponse executeCqlQuery(CqlRequest cqlRequest)
         actionRegistry);
   }
 
+  private void attachFileWithTransformer(
+      ServiceReference<QueryResponseTransformer> queryResponseTransformer,
+      CqlQueryResponse cqlQueryResponse,
+      Request request,
+      Response response) {
+    response.status(500);
+    BinaryContent content;
+    try {
+      content =
+          bundleContext
+              .getService(queryResponseTransformer)
+              .transform(cqlQueryResponse.getQueryResponse(), new HashMap<>());
```

Comment:
You're sending an empty map as the second parameter to this query response transformer. That means you're not sending information about Hidden Fields, Order, or Aliases. 

Please see the existing CsvQueryResponse transformer to see what I mean: 

https://github.com/codice/ddf/blob/master/catalog/transformer/catalog-transformer-csv-queryresponse/src/main/java/ddf/catalog/transformer/csv/CsvQueryResponseTransformer.java#L93